### PR TITLE
starting documentation

### LINKED ISSUE #9: Documentation
What's the plan for documentation? Readme? Inline in the gh-pages? Are we going to have to do anything different for components that are mostly from 3rd party libraries?

### PR DESCRIPTION
**Pull request by gvn**
closes #9

### CODE REVIEW

**Comment by alanmoo** [comment]
This is fine for now, creating another ticket for better distribution workflow

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+### File Structure
+
+```
+├── demo <- A page with all components for dev and demo purposes
+│   ├── css <- Compiled code. Don't edit.
+│   ├── img <- Image assets for demo page.
+│   ├── index.html <- Wrapper for demo.jsx injection
+│   └── js <- Compiled code. Don't edit.
+├── dist <- Compiled code. Don't edit.
```
Comment (alanmoo) [inline README.md]: Update this line since you added .gitignore?

**Comment by alanmoo** [comment]
Uh, scratch that, you snuck the changes in.

File: README.md (hunk 1 above)
Comment (gvn) [inline README.md]: I think it's still good to have there regardless.

**Comment by gvn** [comment]
So is this good to merge then?

**Comment by alanmoo** [comment]
Yeah

**Pull request merged**